website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js

```diff
@@ -51,4 +51,9 @@ app.controller('MainController',
         var cmdlet = productInfo[$routeParams.product][$routeParams.cmdlet];
         return cmdlet.parameters;
       };
+      
+      /* Tells us if the object passed in is empty */
```

Comment:
When you have a multi-line comment, it is a matter of preference to use a single multi-line comment block, or multiple single-line comments. (I personally prefer the former when Closure type annotations are used, and the latter for everything else.)

However, if you only have a single-line comment. (Like this one.) You should always use a single-line comment for it.